docs/content/design/kubernetes-mixin.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Kuberentes Mixin 
+description: The Porter Kubernetes Mixin 
+---
+
+# Kubernetes Mixin
+
+## Overview
+
+The Kubernetes Mixin provides bundle authors with the ability to apply and delete Kubernetes manifests. The mixin will leverage `kubectl`, similar to how the Helm mixin utilizes the `helm` command line tool.
+
+## Build Time
+
+## Dry Run
+
+The mixin will use the `kubectl create` dry-run flag in order to perform a dry run for the bundle.
```

Comment:
Can we use the `kubectl apply --dry-run` flag instead of the create flag for consistency with what the runtime will actually do?